lib/modulr/api/transactions_service.rb

```diff
@@ -3,10 +3,31 @@
 module Modulr
   module API
     class TransactionsService < Service
-      def history(account_id:)
-        response = client.get("/accounts/#{account_id}/transactions")
+      def list(account_id:, **opts)
+        response = client.get("/accounts/#{account_id}/transactions", build_query_params(opts))
         Resources::Transactions::Transactions.new(response, response.body[:content])
       end
+
+      private def build_query_params(opts) # rubocop:disable Metrics/AbcSize
+        same_name_params = [:id, :amount, :currency, :description, :credit, :type]
+        transactions_date_params = { to: :toTransactionDate, from: :fromTransactionDate }
+        # posted_date_params = { to: :toPostedDate, from: :fromPostedDate}
+        # amount_params = {to: :minAmount, from: :maxAmount}
+        mapped_params = {
```

Comment:
Some of these fields are not allowed in the [Transactions endpoint](https://modulr.readme.io/reference/gettransactionsbyaccount):
- transactionDate
- postedDate
- sourceExternalReference
- additionalInfo